ui/src/app/components/prepare-migration/prepare-migration.component.html

```diff
@@ -238,12 +251,20 @@ <h3> Generated Resources:</h3>
             </li>
         </span>
     </div>
+    <div *ngIf="isDataprocJobsGenerated">
+        <span *ngIf="selectedMigrationType ==='dproc' && dataprocJobsGenerated.DataprocJobUrls != null && dataprocJobsGenerated.DataprocJobUrls.length > 0">
```

Comment:
Please verify once that dataprocJobsGenerated.DataprocJobUrls is coming as null or undefined and compare using !==

Reply:
`dataprocJobsGenerated.DataprocJobUrls` will neither come as null nor undefined, it's default value is `[]`. Hence, removed the unecesaary check
Commit: https://github.com/cloudspannerecosystem/harbourbridge/pull/519/commits/6e88d25c184e1071a365d97f58ae39dbf28c7702